.benchmark.sh

```diff
@@ -5,7 +5,7 @@ set -e +o pipefail
 if [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
     echo -e "NOTE: The CPU benchmarks are performed on Travis VMs and vary widly between runs," > .benchmark.body
     echo -e " you can't trust them. The memory benchmarks are OK\n\n" >> .benchmark.body
-    cat .benchmark.log >> .benchmark.body
+    cat .benchmark.log | awk '/^benchmark old/ { printf "%s\n%s\n", "```", $0 }; /^$/ { print "```" }; /^Bench/ { print $0 }; END{ print "```" }' >  .benchmark.body
```

Comment:
had the same thought :)

However you need a `>>` instead of `>` because now you're overwritting .benchmark.body ?

Reply:
whoops.